packages/devtools_app/pubspec.lock

```diff
@@ -827,5 +827,5 @@ packages:
     source: hosted
     version: "2.2.1"
 sdks:
-  dart: ">=2.10.0-110 <=2.11.0-229.0.dev"
+  dart: ">=2.10.0-110 <=2.11.0-256.0.dev"
```

Comment:
should we loosen this dart sdk constraint to be before 3.0?

Reply:
I think the lock file just automatically updates to whatever version of flutter you are on?